controllers/policyserver_controller.go

```diff
@@ -191,6 +207,28 @@ func (r *PolicyServerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				},
 			}
 		})).
+		Watches(&source.Kind{Type: &corev1.Secret{}}, handler.EnqueueRequestsFromMapFunc(func(object client.Object) []reconcile.Request {
+			// This watch ensures that if the policy server CA
```

Comment:
If this is the watch for PolicyServer CA secrets, are we missing a watch for the Kubewarden CA secret?
Could it be that without that second watch we only create the Kubewarden CA secret on controller initalization? If so, is that enough?

Reply:
The CA controller ensures that the root CA will be there and all the policy server secrets will update when the root CA changed.

This question is also related to my comment [here](https://github.com/kubewarden/kubewarden-controller/pull/502/files#r1290656323)